src/StackAdmin/Resources/Commands.Resources/Commands.Resources.csproj

```diff
@@ -0,0 +1,179 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
use the $(LibrarySourceFolder) variable instead of ..\..\..\ once the common targets are open

Reply:
using stackpackagefolder and librarysourcefolder now